configurations/dockerFileTemplate.go

```diff
@@ -8,57 +8,54 @@ import (
 )
 
 type dockerfileTemplate struct {
-	Data *dockerfileData
+	Data serverConfig
 }
 
 // Gives a new instance of dockerfile template.
-func newDockerfileTemplate(data *dockerfileData) *dockerfileTemplate {
+func newDockerfileTemplate(data serverConfig) *dockerfileTemplate {
 	return &dockerfileTemplate{Data: data}
 }
 
-// Unmarshal yaml file and gives an instance of dockerfileData.
-func newDockerFileDataFromYamlFile(filename string) (*dockerfileData, error) {
+// Unmarshal yaml file and gives an instance of serverConfig.
+func newDockerFileDataFromYamlFile(filename string) (serverConfig, error) {
 	node := yaml.Node{}
 
 	err := unmarshalYamlFile(filename, &node)
 	if err != nil {
 		return nil, fmt.Errorf("unmarshal: %v", err)
 	}
 
-	stages, err := getStagesDataFromNode(node.Content[0])
+	stageData, err := getStagesDataFromNode(node.Content[0])
```

Comment:
again `stage` is back!!
What is `stageData`?

Reply:
Changed